addons/modules/stringtable.xml

```diff
@@ -448,111 +487,127 @@
             <Japanese>縦隊</Japanese>
             <Chinese>單縱隊形</Chinese>
             <Chinesesimp>单纵队形</Chinesesimp>
             <Russian>Колонна</Russian>
+            <German>Kolonne</German>
         </Key>
         <Key ID="STR_CBA_Modules_StagColumn">
             <English>Stag Column</English>
             <Japanese>千鳥縦隊</Japanese>
             <Chinese>雙縱隊形</Chinese>
             <Chinesesimp>双纵队形</Chinesesimp>
             <Russian>Колонна в шахматном порядке</Russian>
+            <German>Gestaffelte Kolonne</German>
         </Key>
         <Key ID="STR_CBA_Modules_Wedge">
             <English>Wedge</English>
             <Japanese>傘型隊形</Japanese>
             <Chinese>楔形</Chinese>
             <Chinesesimp>楔形</Chinesesimp>
             <Russian>Клин</Russian>
+            <German>Keil</German>
         </Key>
         <Key ID="STR_CBA_Modules_EchLeft">
             <English>Ech Left</English>
             <Japanese>左梯形</Japanese>
             <Chinese>左梯隊形</Chinese>
             <Chinesesimp>左梯队形</Chinesesimp>
             <Russian>Левый эшелон</Russian>
+            <German>Staffel Links</German>
         </Key>
         <Key ID="STR_CBA_Modules_EchRight">
             <English>Ech Right</English>
             <Japanese>右梯形</Japanese>
             <Chinese>右梯隊形</Chinese>
             <Chinesesimp>右梯队形</Chinesesimp>
             <Russian>Правый эшелон</Russian>
+            <German>Staffel Rechts</German>
         </Key>
         <Key ID="STR_CBA_Modules_Vee">
             <English>Vee</English>
             <Japanese>V字隊形</Japanese>
             <Chinese>V字隊形</Chinese>
             <Chinesesimp>V字队形</Chinesesimp>
             <Russian>V-образный</Russian>
+            <German>V-förmig</German>
         </Key>
         <Key ID="STR_CBA_Modules_Line">
             <English>Line</English>
             <Japanese>横隊</Japanese>
             <Chinese>一字隊形</Chinese>
             <Chinesesimp>一字队形</Chinesesimp>
             <Russian>Линия</Russian>
+            <German>Linie</German>
         </Key>
         <Key ID="STR_CBA_Modules_File">
             <English>File</English>
             <Japanese>縦列</Japanese>
             <Chinese>緊湊縱隊</Chinese>
             <Chinesesimp>紧凑纵队</Chinesesimp>
             <Russian>Ряд</Russian>
+            <German>In Reih und Glied</German>
         </Key>
         <Key ID="STR_CBA_Modules_Diamond">
             <English>Diamond</English>
             <Japanese>ダイヤモンド</Japanese>
             <Chinese>菱形隊形</Chinese>
             <Chinesesimp>菱形队形</Chinesesimp>
             <Russian>Ромб</Russian>
+            <German>Raute</German>
         </Key>
         <Key ID="STR_CBA_Modules_ExecutableCode">
             <English>Code to Execute</English>
             <Japanese>コードの実行</Japanese>
             <Chinese>會執行的腳本代碼</Chinese>
             <Chinesesimp>会执行的脚本代码</Chinesesimp>
             <Russian>Код для Выполнения</Russian>
+            <German>Programmcode ausführen</German>
```

Comment:
`Auszuführender Programmcode` would fit better in my opinion

Reply:
Minus "Programm-"